firefox-ios/Client/Glean/probes/onboarding.yaml

```diff
@@ -421,4 +421,29 @@ onboarding:
     notification_emails:
       - fx-ios-data-stewards@mozilla.com
     expires: "2026-01-01"
+  go_to_settings_pressed:
+    type: counter
+    description: |
+      Counts the number of times the Go To Settings button on
+      default browser onboarding is clicked. This applies to both
+      legacy and modern onboarding flows.
+    bugs:
+      - https://github.com/mozilla-mobile/firefox-ios/issues/31366
+    data_reviews:
```

Comment:
The previous data reviews should carry over from the removed files. The bugs is not as important.